region/wreckedship/main.json

```diff
@@ -2206,7 +2233,35 @@
                   "name": "Base",
                   "notable": false,
                   "requires": [ "h_canBombThings" ],
-                  "note": "Those bomb blocks respawn. They must be broken on the way back too."
+                  "note": "The bomb block respawns. It must be broken on the way back too."
+                },
+                {
+                  "name": "SpeedKeep Bounce and SpringBall",
+                  "notable": false,
+                  "requires": [
+                    "h_canUseSpringBall",
+                    "canSpeedKeep",
```

Comment:
should also require phantoon killed.

Reply:
or possibly a speedkeep chain? I'm not sure if you can get over the pits that way with this low ceiling